common/components/chrome/MainHeader.jsx

```diff
@@ -1,226 +1,336 @@
-          </Drawer>
-
-          <DropdownButton
-            className="MainHeader-dropdownButton"
-            bsStyle="link"
-            title={CurrentUser.firstName() + ' ' + CurrentUser.lastName()}
-            noCaret
-            id="dropdown-no-caret"
-          >
-            {this._renderFooterMenuLinks()}
-            <MenuItem divider />
-            <MenuItem href="/logout/">Logout</MenuItem>
-          </DropdownButton>
-        </React.Fragment>
-
-      )
-      : (
-        <React.Fragment>
-
-          <div className="MainHeader-hamburger" onClick={() => this._toggleDrawer('left', true)}>
-            <IconButton >
-              <MenuIcon />
-            </IconButton>
           </div>
-
-          <Drawer open={this.state.left} onClose={() => this._toggleDrawer('left', false)}>
-            <div
-              tabIndex={0}
-              role="button"
-              onClick={() => this._toggleDrawer('left', false)}
-              onKeyDown={() => this._toggleDrawer('left', false)}
-            >
-              <div className="MainHeader-navDrawer">
-                {this._renderFooterDrawerLinks()}
-                <div className={'MainHeader-drawerDiv'} onClick={this._onLogInClick}>
-                  <a href="" > Log In </a>
-                </div>
-                <Divider />
-                <div className={'MainHeader-drawerDiv'} onClick={this._onSignUpClick} >
-                  <a href="" > Sign Up </a>
-                </div>
-                <Divider />
-                <div className={'MainHeader-drawerDiv'} onClick={this._onResetPasswordClick} >
-                  <a href="" > Forgot Password </a>
-                </div>
-              </div>
-            </div>
-          </Drawer>
-
-          <span className="MainHeader-links">
-            {this._renderFooterLinks()}
-            <span onClick={this._onLogInClick}>
-              <a href="" > Log In </a>
-            </span> |{' '}
-
-            <span onClick={this._onSignUpClick} >
-              <a href="" > Sign Up </a>
-            </span> |{' '}
-
-            <span onClick={this._onResetPasswordClick} >
-              <a href="" > Forgot Password </a>
-            </span>
-          </span>
-        </ React.Fragment>
-      );
-  }
-
-  _toggleDrawer(side: string, open: boolean): void {
-    this.setState({ [side]: open })
+        </Drawer>
+      </React.Fragment>
+    
+    )
   }
-
-  _renderFooterLinks(): $ReadOnlyArray<React$Node> {
-    return FooterLinks.list().map((link, i) => {
+  
+  _renderHamburgerFooterLinks(): $ReadOnlyArray<React$Node> {
+    return FooterLinks.list().map((link) => {
       return (
-        <span key={i}>
-          <a
-            href={link.url}
-            target="_blank" rel="noopener noreferrer"
-            onClick={FooterLinks.logClick.bind(this, link)}
-          >
-            {link.name}
+        <React.Fragment>
+          <a key={link.url} href={link.url} target="_blank" rel="noopener noreferrer">
```

Comment:
Since we've fixed the double-entry in the history bug, I think we can stop using target="_blank" here, the back button works.